infra/ansible/roles/build_srcs/tasks/main.yaml

```diff
@@ -17,18 +17,6 @@
     name: "{{ pytorch_wheels.files | map(attribute='path') }}"
     state: "forcereinstall"
 
-- name: Check if build_torch_xla_libs.sh script exists
```

Comment:
This step can be removed as the bash script was deleted long time ago. This is the output of the recent trigger run: https://screenshot.googleplex.com/AEsb2my8yBKbG8h

Reply:
Good catch!